testing/src/cypress/page-objects/Institution-objects/ManageUsersObject.ts

```diff
@@ -3,30 +3,50 @@ export default class ManageUsersObject {
     return cy.contains("Manage Users");
   }
 
-  userSummaryMessage() {
-    return cy.contains("User Summary");
+  manageUsersSummary() {
+    return cy.contains("Manage Users");
   }
 
   editButtonFirstRow() {
-    return cy.get("[data-icon='pen']").eq(0);
+    return cy.get("[data-cy='editUser']").eq(0);
```

Comment:
To my knowledge, when the `.eq(0)` is required? Is it required for any `v-btn`?

Reply:
`.eq` is essentially locating the element by index. In this case its the first element with that is found by that attribute. I updated not to use this method.